src/TableEditColumn.js

```diff
@@ -179,6 +179,7 @@ class TableEditColumn extends Component {
     let { className } = this.state;
     // put placeholder if exist
     editable.placeholder && (attr.placeholder = editable.placeholder);
+    editable.attrs && (attr = {...editable.attrs, ...attr});
```

Comment:
Yeah that's easier :grin: I spread the user supplied attrs first though so the ref and handlers can't be overwritten.